modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java

```diff
@@ -458,6 +467,19 @@ private void _deleteSegmentsExperiment(
 			segmentsExperiment, ResourceConstants.SCOPE_INDIVIDUAL);
 	}
 
+	private int _getLowestPriority(
+		long groupId, long classNameId, long classPK) {
+
+		return Optional.ofNullable(
+			segmentsExperiencePersistence.fetchByG_C_C_A_Last(
+				groupId, classNameId, classPK, true, null)
```

Comment:
If only the active experiences are taking into account, it would occur an error after creating a variation in an AB Testing. The lowest method should not filter by active

Reply:
![image](https://user-images.githubusercontent.com/10129814/104699079-7e01b100-5712-11eb-8f9d-1c050a3aff33.png)

![image](https://user-images.githubusercontent.com/10129814/104699120-8bb73680-5712-11eb-8902-a309ee56c00e.png)

![image](https://user-images.githubusercontent.com/10129814/104699189-a4bfe780-5712-11eb-9d6d-d06284b06deb.png)

Reply:
Because is trying to set an already used priority

Reply:
This is the bug we talked about with @dgarciasarai !!!

Please, in this review, just take a look to the backend, check that the DB works property, forget about the frontend part.

Reply:
Please, could you describe the steps, I don't understand the screenshots :(

Reply:
Sure!

1. Create a page and publish
2. Create an AB Test and a variant
3. Edit the page and try to create a new experience (nevertheless the segment)

Reply:
Previously it was used the following method to get the new priority:

´´´java
        @Override
	public int getSegmentsExperiencesCount(
		long groupId, long classNameId, long classPK) {

		return segmentsExperiencePersistence.countByG_C_C(
			groupId, classNameId, _getPublishedLayoutClassPK(classPK));
	}
```	
not filtering by active

Reply:
The error happens because there's a constraint in the servcice.xml

```xml
                 <finder name="G_C_C_P" return-type="SegmentsExperience" unique="true">
			<finder-column name="groupId" />
			<finder-column name="classNameId" />
			<finder-column name="classPK" />
			<finder-column name="priority" />
		</finder>
``` 
The uniqueness of an experience does not take into account the active field, so if we filter by it creating experiences, it is possible that this priority is already in use.

Reply:
@cgoncas, Is it clear know?

Reply:
Great, it is just enough with the steps. Thanks!!

Reply:
Hey @darquesdev !!

What should be the position of the new A/B Testing experience?. It should be on top of all the experiences? Like before?

Thanks!!

Reply:
It does not matter. 

Currently, if you try to create 2 variants of an AB Test there's an error and the expected behavior is that you can create more than one variant.

Reply:
We have decided that we will create de A/B Testing experiences in the top of the list.